### PR TITLE
revamp modal close button

### PR DESCRIPTION
**Pull request by ngilain**
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1231
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/88

### CODE REVIEW

File: MediaAdminBundle/Resources/views/BackOffice/Underscore/mediaModalView._tpl.twig [hunk 1]
```diff
@@ -1,9 +1,18 @@
-<div class="modal fade mediaModalContainer divMessageBox" tabindex="-1" role="dialog" aria-labelledby="" aria-hidden="true" data-backdrop="static" data-input="<%= input %>">
+<div
+    class="modal fade mediaModalContainer
+    divMessageBox" tabindex="-1" role="dialog"
```
Comment (ghost) [inline MediaAdminBundle/Resources/views/BackOffice/Underscore/mediaModalView._tpl.twig]: Are you sure that a new line in the middle of an HTML attribute value is working well ?

**Comment by ngilain** [comment]
@itkg-canne : update done

File: MediaAdminBundle/Resources/views/BackOffice/Underscore/mediaModalView._tpl.twig [hunk 2]
```diff
@@ -1,9 +1,18 @@
...
+    tabindex="-1" role="dialog"
+    aria-labelledby="" aria-hidden="true"
+    data-backdrop="static" data-input="<%= input %>"
+>
     <div class="modal-dialog">
         <div class="modal-content">
             <div class="modal-header">
-                <button type="button" class="close mediaModalClose"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
-                <h4 class="modal-title">{{ 'open_orchestra_media_admin.left_menu.editorial.media'|trans }}</h4>
+                <button type="button" class="btn btn-success mediaModalClose" data-dismiss="modal">
```
Comment (nicolasThal) [inline MediaAdminBundle/Resources/views/BackOffice/Underscore/mediaModalView._tpl.twig]: same btn-warning instead

**Comment by ghost** [comment]
:+1:

**Comment by ngilain** [comment]
@nicolasThal : update done

**Comment by veolys-nanne** [comment]
:+1:

**Pull request merged**